### PR TITLE
ARARE

### PR DESCRIPTION
**Pull request by Doenyas**
ARARE

### CODE REVIEW

File: xspo-alliance-members.json [hunk 1]
```diff
@@ -42,7 +42,13 @@
                 "pool_id": "5cff121b7239ea76035bcb79711674a29fdf35d0bbe83a212467fab9",
                 "member_since": "2021-07-20",
                 "name": "SPIDR (SpidarPool)"
+            },
+            "4": {
+                "pool_id": "pool17etnr2ngwpnqwuvdzszzd36u67kzcsrs8scuxf0a63hezy9zvpv",
```
Comment (ResponsibleStaking) [inline xspo-alliance-members.json:47]: Please use the Bech32 ID of your pool

**Review by ResponsibleStaking — CHANGES_REQUESTED** [review]
Please update the pool ID

**Review by Doenyas — COMMENTED** [review]
updated pool id

File: xspo-alliance-members.json (hunk 1 above)
Comment (Doenyas) [reply xspo-alliance-members.json]: Done.

**Review by Golden-VS — APPROVED** [review]

**Pull request merged**